frontend/coprs_frontend/coprs/auth.py

```diff
@@ -22,38 +19,48 @@ class UserAuth:
     encapsulated within this class.
     """
 
-    @staticmethod
-    def logout():
+    @classmethod
+    def next_url(cls):
+        """
+        Where should user be redirected after a successful login?
+
+        At this moment, we redirect to the user profile but it would also make
+        sense to redirect back at the page where user clicked the login button.
+
+        We used to get the next URL through `oid.get_next_url()` but there is
+        no such equivalent for our current OIDC client. We need to set and get
+        it manually.
```

Comment:
If there really isn't a support for referrer, we probably need to handle this via a "hop" pre-login page ourselves?